internal/controlplane/handlers_githubwebhooks.go

```diff
@@ -869,9 +881,19 @@ func reconcilePrWithDb(
 
 	switch prEvalInfo.Action {
 	case WebhookActionEventOpened, WebhookActionEventSynchronize:
+		externalID, err := proto.Marshal(&pb.GitHubPullRequestID{
```

Comment:
Ideally we would have a `PullRequestService` which encapsulates methods for creating these entities. I am avoiding this sort of refactoring until we have the provider/core code more cleanly split.